src/analyses/call_graph.cpp

```diff
@@ -18,14 +18,9 @@ call_grapht::call_grapht()
 {
 }
 
-call_grapht::call_grapht(const goto_modelt &goto_model):
-  call_grapht(goto_model.goto_functions)
+call_grapht::call_grapht(const goto_modelt &goto_model)
 {
-}
-
-call_grapht::call_grapht(const goto_functionst &goto_functions)
-{
```

Comment:
Why remove this constructor?

Reply:
Ah, this is left over from call graph changes. Will revert